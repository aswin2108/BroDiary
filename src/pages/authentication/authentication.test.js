import renderer from 'react-test-renderer';
import Authentication from './authentication';

it('Authentication page rendered correctly', ()=>{
    const authPage=renderer.create(
            <Authentication/>
        ).toJSON();
      expect(authPage).toMatchSnapshot();
});