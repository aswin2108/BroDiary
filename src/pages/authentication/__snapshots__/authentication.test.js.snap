// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Authentication page rendered correctly 1`] = `
<div
  className="authentication-container"
>
  <div
    className="sign-in-container"
  >
    <h2
      className="sign-in-title"
    >
      Already have an account?
    </h2>
    <span
      className="sign-in-sub-title"
    >
      Sign in with your email and password
    </span>
    <form
      onSubmit={[Function]}
    >
      <div
        className="group"
      >
        <input
          className="form-input"
          name="email"
          onChange={[Function]}
          required={true}
          type="email"
          value=""
        />
        <label
          className="
                  form-input-label"
        >
          Email
        </label>
      </div>
      <div
        className="group"
      >
        <input
          className="form-input"
          name="password"
          onChange={[Function]}
          required={true}
          type="password"
          value=""
        />
        <label
          className="
                  form-input-label"
        >
          Password
        </label>
      </div>
      <div
        className="buttons-container"
      >
        <button
          className="button-container undefined"
          type="submit"
        >
          Sign In
        </button>
        <button
          className="button-container google-sign-in"
          onClick={[Function]}
          type="button"
        >
          Google Sign In
        </button>
      </div>
    </form>
  </div>
  <div
    className="sign-up-container"
  >
    <h2
      className="sign-up-title"
    >
      Don't have an account?
    </h2>
    <span
      className="sign-up-sub-title"
    >
      Sign up with your email and password
    </span>
    <form
      onSubmit={[Function]}
    >
      <div
        className="group"
      >
        <input
          className="form-input"
          name="displayName"
          onChange={[Function]}
          required={true}
          type="text"
          value=""
        />
        <label
          className="
                  form-input-label"
        >
          Display Name
        </label>
      </div>
      <div
        className="group"
      >
        <input
          className="form-input"
          name="email"
          onChange={[Function]}
          required={true}
          type="email"
          value=""
        />
        <label
          className="
                  form-input-label"
        >
          Email
        </label>
      </div>
      <div
        className="group"
      >
        <input
          className="form-input"
          name="password"
          onChange={[Function]}
          required={true}
          type="password"
          value=""
        />
        <label
          className="
                  form-input-label"
        >
          Password
        </label>
      </div>
      <div
        className="group"
      >
        <input
          className="form-input"
          name="confirmPassword"
          onChange={[Function]}
          required={true}
          type="password"
          value=""
        />
        <label
          className="
                  form-input-label"
        >
          Confirm Password
        </label>
      </div>
      <button
        className="button-container undefined"
        type="submit"
      >
        Sign Up
      </button>
    </form>
  </div>
</div>
`;
